tectonic/core-bridge.c

```diff
@@ -69,7 +71,7 @@ dvipdfmx_simple_main(tt_bridge_api_t *api, char *dviname, char *pdfname)
 {
     extern int dvipdfmx_main(int argc, char *argv[]);
 
-    char *argv[] = { "dvipdfmx", "-o", pdfname, dviname };
+    char *argv[] = { xstrdup("dvipdfmx"), xstrdup("-o"), pdfname, dviname };
```

Comment:
`const char*` was being assigned to `char*`. So there is a potential that the program might try to change the static strings.
Actually, wouldn't just declaring `argv` with `const char *argv[]` solve this since `argv` is not used to mutate the content in the code as far as I can tell? This would need also `dvipdfmx_main` to be declared with `const char* argv`, or would just casting be enough like in `dvipdfmx_main(4, (char**) argv)` (I'm not sure if this is the correct syntax for the cast)?